sdk-springboot/src/main/java/io/dapr/springboot/DaprSubscriptionBuilder.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright (c) Microsoft Corporation and Dapr Contributors.
+ * Licensed under the MIT License.
+ */
+
+package io.dapr.springboot;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class DaprSubscriptionBuilder {
```

Comment:
Does this really need to be public? Make it package visible if possible. It is only used in a private property.